### DISCUSSION TITLE
Cables not deleted after device deletion

### DISCUSSION THREAD

**Discussion by jayGG83** [netbox-community/netbox#14093] [Q&A] [answered]
Hello,

Before creating a bug/feature, I'd like to ask her first.
After deleting some devices that has cable connection between interfaces, the cable itself is not deleted automatically. Is this the correct process or its a bug?

Netbox version: 3.6.3

Thanks

**Comment by jeremystretch** [comment, marked as answer]
This is an open discussion under #10769. At this stage, I think we recognize that a change is needed, but we're still determining the appropriate fix.

**Reply by jayGG83** [reply]
OK, closing this then. Thanks

**Reply by jayGG83** [reply]
An aproach from a previous project I was is that, in case there are active connections between two devices, an error message informing about the active cable was shown impeding the deletion. User must delete first the cable in order to delete the device
